packages/frontend/src/hooks/useSavedQuery.ts

```diff
@@ -182,27 +182,34 @@ export const useChartVersionRollbackMutation = (
 export const useSavedQueryDeleteMutation = () => {
     const queryClient = useQueryClient();
     const { showToastSuccess, showToastError } = useToaster();
-    return useMutation<undefined, ApiError, string>(deleteSavedQuery, {
-        mutationKey: ['saved_query_create'],
-        onSuccess: async () => {
-            await queryClient.invalidateQueries('spaces');
-            await queryClient.invalidateQueries('space');
-            await queryClient.invalidateQueries('pinned_items');
-            await queryClient.invalidateQueries(
-                'most-popular-and-recently-updated',
-            );
+    return useMutation<undefined, ApiError, string>(
+        async (data) => {
+            await queryClient.removeQueries(['savedChartResults', data]);
```

Comment:
`invalidateQueries` did not work here? also isn't it better to do in the onSuccess ?

Reply:
I've tried with invalidateQueries and it didn't work, see my thread : https://lightdash.slack.com/archives/C02GQKJK84Q/p1701937791955869

I'm doing it here instead of `onSuccess` because I don't have access to the `chartUuid` (response from delete is empty) . 

Let me know if you have a better way of doing this.

Reply:
ah that makes sense. so data in that case is UUID